packages/flame_3d/lib/src/resources/material/spatial_material.dart

```diff
@@ -61,18 +60,18 @@ class SpatialMaterial extends Material {
 
   @override
   void bind(GraphicsDevice device) {
+    final lightPosition = device.lightPosition;
+    if (lightPosition == null) {
+      return;
+    }
```

Comment:
Instead of returning we should maybe do `?? Vector3.zero()` because now a mesh wont have a material if someone didn't add a light.

Or better (and a follow-up PR) have a default light like godot so that will be used **unless** there is a light in the scene.

Reply:
that would cause us to actually add a light at (0, 0, 0), which would be kinda weird?

I think we can add a default light later when we support an omni light. these are spot lights so far

Reply:
(right now if you don't add any lights nothing is renderer, which kinda makes sense)